src/sss_client/pam_sss_gss.c

```diff
@@ -195,6 +195,8 @@ static errno_t sssd_gssapi_init_send(pam_handle_t *pamh,
     struct sss_cli_req_data req_data;
     size_t service_len;
     size_t user_len;
+    size_t reply_len;
+    uint8_t *reply;
```

Comment:
Hi,

I would suggest initialize `reply` as `NULL`  to make sure `free(reply)` later on is safe even if `reply` is not touched in case of an error.

bye,
Sumit

Reply:
Done.